narwhal/source_code_sharing/update_liferay_dxp_source_code_of_hotfixes.sh

```diff
@@ -32,19 +37,39 @@ function check_if_tag_exists {
 	fi
 }
 
+function check_ignore_argument {
+	if [[ "x${IGNORE_ZIP_FILES}" =~ x*${hotfix_zip_file}* ]]
+	then
+		lc_log DEBUG "Ignoring '${file_url}'."
+
+		return "${LIFERAY_COMMON_EXIT_CODE_OK}"
+	else
+		lc_log DEBUG "The file on '${file_url}' is not on the ignore list."
```

Comment:
$fIle_url is an invalid variable in this function.

'The file on' -> Should be something like 'The file'

'is not on the ignore list.' -> Should be something like 'is not ignored via command line argument'. Something that tells more information about the reason.

Reply:
$fIle_url is is reachable because of parent function